test/integration/guest_can_create_account_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class GuestCanCreateAccountTest < ActionDispatch::IntegrationTest
+
+  test "guest can view the homepage" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.